components/builder-depot/src/server.rs

```diff
@@ -749,18 +755,6 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
             }
         }
 
-        log_event!(
-            req,
-            Event::PackageUpload {
-                origin: ident.get_origin().to_string(),
-                package: ident.get_name().to_string(),
-                version: ident.get_version().to_string(),
-                release: ident.get_release().to_string(),
-                target: target_from_artifact.to_string(),
-                account: session.get_id().to_string(),
-            }
-        );
```

Comment:
I'll admit that I'm not even sure what this event logging is all about.  Are we not using it?

Reply:
No, it was an early stab at an event design, and never got fully implemented.  I think we'll end up re-doing it.